docs-public/snippets.js

```diff
@@ -1005,15 +1007,27 @@ function buildExampleEditorForm(exampleTemplate) {
     return form;
 }
 
-// function getExampleDependencies(directives, production) {
-//     return directives.filter(function(dir) {
-//         return dir.module.indexOf('@progress') === 0 ||
-//                dir.module.indexOf('@telerik') === 0;
-//     }).reduce(function(result, dir) {
-//         result[dir.module] = production ? 'latest' : 'dev';
-//         return result;
-//     }, {});
-// }
+function getExampleDependencies(files, channel) {
+    var imports = {};
+    var dependencyMatcher = /import.*from\s*['|"](.*)['|"]/g;
+    for (var key in files) {
+        if (files.hasOwnProperty(key)) {
+            var found;
+            while ((found = dependencyMatcher.exec(files[key])) !== null) {
+                // tmp[1] represents the group which match the package being imported.
```

Comment:
The comment is incorrect, and describes what the `group` variable means. A better way to achieve this is to find a better name for the variable, e.g. `var importedPackage = found[1];`.